src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java

```diff
@@ -44,6 +45,9 @@
   private static final int DELETION_INTERVAL_MAX = 60;
   private static final String HTTPS_ENABLED = "https-enabled";
   private static final String WRITER_QUEUE_SIZE = "writer-queue-size";
+  private static final String BATCH_METRICS_RETENTION_PERIOD = "batch-metrics-retention-period";
+  private static final long BATCH_METRICS_RETENTION_PERIOD_DEFAULT = 7;  // 7 minutes of metrics
```

Comment:
Also can we add javadoc comments describing the variables ?

Reply:
Will add this documentation to the batchMetricsRetentionPeriod variable (this is what was done for the other parameters also).